eos/utils/kinematic_TEST.cc

```diff
@@ -48,6 +48,30 @@ class KinematicsTest :
                 TEST_CHECK_EQUAL(6.0, kinematics["s_max"]);
             }
 
+            // Creation from initializer list and aliasing
+            {
+                Kinematics kinematics
+                {
+                    { "s_min", 1.0 },
+                    { "s_max", 6.0 },
+                };
+                kinematics.alias("q2_min", "s_min");
+                kinematics.alias("q2_max", "s_max");
+
+                TEST_CHECK_EQUAL(1.0, kinematics["s_min"]);
+                TEST_CHECK_EQUAL(6.0, kinematics["s_max"]);
+                TEST_CHECK_EQUAL(1.0, kinematics["q2_min"]);
+                TEST_CHECK_EQUAL(6.0, kinematics["q2_max"]);
+
+                TEST_CHECK_EQUAL("s_min", kinematics["s_min"].name());
+                TEST_CHECK_EQUAL("s_min", kinematics["s_min"].name());
+
+                // Test clearing the alias map and access values
+                auto saved_alias = kinematics["s_min"];
```

Comment:
``s_min`` is not an alias, it is **aliased*. Please add a test case that saves, in addition, ``q2_min`` and ``q2_max`` and check that they surive clearing the alias map.